resources/lang/en/forms.php

```diff
@@ -72,6 +72,10 @@
         'description'    => 'Manage and logout your active sessions on other browsers and devices.',
```

Comment:
maybe we should add that it only shows the 3 most recent entries 🤔 will discuss with Sam

Reply:
Change the description to `Manage and logout your active sessions on other browsers and devices. Your most recent sessions are shown below.`

Reply:
Changed